drivers/scale_service.go

```diff
@@ -57,7 +57,7 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient *client
 		return http.StatusBadRequest, fmt.Errorf("Invalid service %v", config.ServiceID)
 	}
 
-	if service.Kind != "service" && service.Kind != "loadBalancerService" {
+	if service.Kind != "scalingGroup" && service.Kind != "loadBalancerService" {
```

Comment:
@cjellick Would we want to keep backwards compatibility to for `service` vs. `scalingGroup`? With this change, we'd have to branch our webhook service for the 1.6.x line and master.